packages/node-http-handler/src/set-socket-keep-alive.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { setSocketKeepAlive } from "./set-socket-keep-alive";
+import { EventEmitter } from "events";
```

Comment:
Workspace throws lint error
```suggestion
import { EventEmitter } from "events";

import { setSocketKeepAlive } from "./set-socket-keep-alive";
```